README.md

```diff
@@ -1,356 +1,258 @@
+### Checkpoints
 
-# Civilian Rules
+1. Must have bar gate + sign.
+2. Civs approaching may be searched.
+3. Check license before disarming; holstered pistols allowed.
+4. Turning around near a checkpoint is illegal.
 
-## General Civilian Rules
+### Impounding
 
-1. Civilians can take over a factory, but can not randomly kill everyone who goes there.
+1. Must impound abandoned vehicles.
+2. Vehicles blocking spawns may be impounded immediately.
 
-## Comp Rules 
+### Insurgents
 
-## If you are doing any of the following, you may not ask for compensation
+1. Blufor and Insurgents may KOS, but arrest is preferred.
+2. May arrest insurgents without a witnessed crime.
 
-1. Carrying large sums around and provoking people to kill you, so that you can steal money from their bank.
+---
 
-2. Standing in the road, lying in the road, running into the road to be hit by a car.
+## 🟥 Opfor
 
-3. Punching armed people.
+### General
 
-4. Jumping out of moving vehicles.
+1. Must allow Civ access to land/resources.
+2. Non-lethal methods encouraged but not required.
+3. Cannot tax more than $150,000.
 
-5. Dropping weapons then claiming to be unarmed.
+### Checkpoints
 
-6. Refusing to be robbed or kidnapped.
+(Same as Blufor)
 
-7. Running around in between an ongoing gunfight.
+### Insurgents
 
-8. Driving a police, Opfor, or Insurgent vehicle.
+1. May arrest for terrorism; hold longer for serious crimes.
+2. No camping Insurgent bases.
+3. Cannot request compensation after being killed by Insurgents.
 
-## Vehicle Death Matching
+### Kill/Arrest
 
-1. Civilians are never permitted to hit people with vehicles unless their lives are in immediate danger.
+1. May only kill Civilians in self-defense without RP.
 
-## Stealing Vehicles
+### OSF
 
-1. Civilians can not steal cars in safe zones.
+1. May conduct RP-valid incursions into North (e.g., kidnapping, convoy theft).
+2. No incursions just to fight Blufor.
 
-2. Civilians are allowed to steal armed or armored vehicles, but the police are allowed to use any means necessary to destroy the vehicle. 
+---
 
-## Gang Rules
+## ⚫ Insurgents
 
-1. Gangs are always permitted to kill one another, to steal drugs or territory.
+1. May KOS Blufor and Opfor only.
+2. Must follow their Caliph.
+3. Cannot kill Civilians unless attacked.
+4. Must help Civilians who claim to be Muslim.
+5. Bombing checkpoints = sparingly.
+6. Do not camp checkpoints.
+7. Cannot align with factions.
+8. May rob convoys.
+9. May hire PMCs.
 
-2. If you are in or near a gang area or drug field and a rival gang kills you, you may not claim comp even if you are unarmed.
+---
 
-3. In order to fight in town, or assassinate other gang members in town, gangs must declare a gang war.
+## 🟡 Civilians
 
-## Civilian Terrorism Rules
+### General
 
-1. Civilians are not allowed to commit terrorist acts. If you want to perform terrorist actions, then switch to the Independent faction.
+1. May take over factories, but not kill indiscriminately.
```

Comment:
Better to say `Allowed to take control of factory zones, but cannot KOS others without valid reasoning.`